Dockerfile

```diff
@@ -1,6 +1,18 @@
-FROM --platform=$TARGETPLATFORM gcc:latest as utils-builder
-
-RUN apt-get update && apt-get install -y flex libkeyutils-dev
+# === Stage 1: Build DRBD utilities ===
+FROM --platform=$TARGETPLATFORM gcc:13 as utils-builder
+
+# Install only necessary tools in minimal environment
+# - flex: needed for parsing in DRBD utils build
+# - libkeyutils-dev: required by DRBD
+# - curl, tar: to fetch and extract source
+# - make: to compile tools
+RUN apt-get update && apt-get install -y \
+    flex \
+    libkeyutils-dev \
+    curl \
+    tar \
+    make \
+ && apt-get clean
```

Comment:
That is completely unnecessary. The build container can have as many tools installed as it wants. I guess this is a nicer list then before, so that is fine.